tests/manage/mcg/test_namespace.py

```diff
@@ -0,0 +1,133 @@
+import logging
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs.bucket_utils import (
+    sync_object_directory,
+    verify_s3_object_integrity
+)
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+logger = logging.getLogger(__name__)
+
+
+class TestNamespace(ManageTest):
+    """
+    Test creation of a namespace resource
+    """
+
+    MCG_NS_RESULT_DIR = '/result'
+    MCG_NS_ORIGINAL_DIR = '/original'
+
+    @tier1
+    def test_namespace_resource_creation(self, ns_resource_factory):
```

Comment:
Please add following marker: `@pytest.mark.polarion_id("OCS-2255")`
(you will also have to import pytest)